### PR TITLE
Drop PHP 7.x from composer.json

### PR DESCRIPTION
**Pull request by TimWolla**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

### Description

Spun out of #17 to unbreak the CI in the 2.11.x base branch and to not mix unrelated changes.

### CODE REVIEW

**Comment by Ocramius** [comment]
The PHPCS failure is because we're on an ancient phpcs release here: merging this as-is first.

File: composer.json [hunk 1]
```diff
@@ -16,17 +16,20 @@
...
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "~8.0.0 || ~8.1.0",
         "laminas/laminas-stdlib": "^3.2.1"
     },
     "require-dev": {
-        "laminas/laminas-coding-standard": "~1.0.0",
-        "laminas/laminas-json": "^3.0.0",
-        "laminas/laminas-session": "^2.8.5",
-        "phpunit/phpunit": "^9.3"
+        "laminas/laminas-coding-standard": "~2.4.0",
```
Comment (Ocramius) [inline composer.json]: Ah, this won't fly as-is 😬

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (TimWolla) [reply composer.json]: Shall I drop the latest commit with the coding standard update?

Comment (Ocramius) [reply composer.json]: Either that, or we actually upgrade the codebase to the new CS ruleset (with the exclusion of strict types, depending on how risky that part is)

Comment (TimWolla) [reply composer.json]: Okay, I've dropped that commit for now.

Comment (Ocramius) [reply composer.json]: We should still aim for a green build, before tagging, btw.

**Comment by Ocramius** [comment]
Thanks @TimWolla!

**Pull request merged**